src/js_parser.zig

```diff
@@ -6643,7 +6643,8 @@ fn NewParser_(
                         var is_sloppy_mode_block_level_fn_stmt = false;
                         const original_member_ref = value.ref;
 
-                        if (symbol.kind == .hoisted_function) {
+                        // TODO: enable this when not bundling. the name for the new symbol needs to go through a renamer
+                        if (p.options.bundle and symbol.kind == .hoisted_function) {
```

Comment:
we run it through the renamer when `--minify-syntax`

maybe add a `p.willRenameSymbols()`?

Reply:
added `willUseNumberRenamer()`. also I should've been more specific, it needs to go through the number renamer.